components/audio_recorder/include/recorder_sr.h

```diff
@@ -64,6 +64,7 @@ typedef struct {
     int          rb_size;                               /*!< Ringbuffer size of recorder sr */
     char         *partition_label;                      /*!< Partition label which stored the model data */
     char         *mn_language;                          /*!< Command language for multinet to load */
+    char         *wn_wakeword;                          /*!< Wake Word for WakeNet to load */
```

Comment:
It is necessary to explain the meaning of NULL in the comments. NULL means to use the first found model,  and this configuration is useful when multi-wakewords selected.